src/main/kotlin/io/github/kryszak/gwatlin/http/serializers/JsonUtil.kt

```diff
@@ -0,0 +1,17 @@
+package io.github.kryszak.gwatlin.http.serializers
+
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+/**
+ * A convenience object for [Json], and to help the
+ * library to use the same instance of it throughout.
+ */
+object JsonUtil {
```

Comment:
I'm not a big fan of `*Util` class naming. In this case the object seems like `JsonConfigurer` maybe? I'm not forcing rename, just thinking out loud :)